src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIJsonSerializerContext.cs

```diff
@@ -1,12 +1,14 @@
 ﻿#if NET6_0_OR_GREATER
 using System;
+using System.Collections.Generic;
 using System.Text.Json;
 using System.Text.Json.Nodes;
 using System.Text.Json.Serialization;
 
 namespace Swashbuckle.AspNetCore.SwaggerUI;
 
 [JsonSerializable(typeof(ConfigObject))]
+[JsonSerializable(typeof(List<UrlDescriptor>))]
 [JsonSerializable(typeof(InterceptorFunctions))]
```

Comment:
Nit: sorting.

Reply:
Put after InterceptorFunctions, let me know if you'd rather sort by the interior type (UrlDescriptor) and I can move it further down.